README.md

```diff
@@ -5,8 +5,8 @@ The Versioneer
 * https://github.com/python-versioneer/python-versioneer
 * Brian Warner
 * License: Public Domain (Unlicense)
-* Compatible with: Python 3.8, 3.9, 3.10, 3.11 and pypy3
-* Experimental support for Python 3.12.
+* Compatible with: Python 3.9, 3.10, 3.11 and pypy3
+* Experimental support for Python 3.12 and 3.13
```

Comment:
I'd say support for 3.12 has been established, and there's no particular reason to call 3.13 experimental once it passes.

```suggestion
* Compatible with: Python 3.9+ and pypy3
```